src/styles/globals.css

```diff
@@ -85,6 +90,34 @@ body {
   .gradient-button-hover {
     background: linear-gradient(0deg, rgba(0, 0, 0, 0.3), rgba(0, 0, 0, 0.3)), linear-gradient(72.57deg, #EC008C 17.15%, #5B10FF 89.41%, #0468D6 102.56%);
   }
+
+  .header-explore-card-bg {
```

Comment:
any reason to have these styling in globals?

Reply:
to have the transition on hover for the card bg will need to add these to globals.css

Reply:
can we add them directly on the element?